recipe/run_test.sh

```diff
@@ -3,5 +3,11 @@ export OMPI_MCA_plm=isolated
 export OMPI_MCA_rmaps_base_oversubscribe=yes
 export OMPI_MCA_btl_vader_single_copy_mechanism=none
 
-python -c "import petsc4py"
-python -c "import petsc4py.PETSc"
+# aarch64 failing tests
+# /lib64/libm.so.6: version `GLIBC_2.27' not found
```

Comment:
if this step is failing, does that mean the builds are broken and should be skipped?

Reply:
Builds are not broken. `libcurand` requires glibc 2.27 symbols, which are usually present where you run, but not on the docker images. Either they don't care, or it hasn't been tested properly yet.

Reply:
@minrk Stefano had already tried running under Docker with emulated Ubuntu 18.04 (what NVIDIA officially supports), and everything is working just fine.